Firebase/InstanceID/FIRInstanceID.m

```diff
@@ -117,6 +118,8 @@ @interface FIRInstanceID ()
 @property(atomic, readwrite, assign) BOOL isFetchingDefaultToken;
 @property(atomic, readwrite, assign) BOOL isDefaultTokenFetchScheduled;
 @property(nonatomic, readwrite, assign) NSInteger retryCountForDefaultToken;
+@property(atomic, strong, nullable)
+    FIRInstanceIDCombinedHandler<NSString *> *defaultTokenFetchHandler;
```

Comment:
I don't think the issue is the default token fetcher, it's simply that one of the return ignore the handler. And also there's always one handler passing in the API so we don't need to keep track of the handlers. Sorry I should've looked at ryan's comment before assigning to you.